examples/data_profiler_demo.ipynb

```diff
@@ -435,6 +439,31 @@
    ]
   },
   {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8e713fc2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load data\n",
+    "data = dp.Data(os.path.join(data_folder, \"csv/diamonds.csv\"))\n",
+    "\n",
+    "# Generate a profile\n",
+    "profile = dp.Profiler(data)\n",
+    "\n",
```

Comment:
no need to load data and save again. otherwise LGTM

Reply:
fine....